gulpfile.js

```diff
@@ -182,44 +182,24 @@ function watchSass() {
 };
 
 function buildSprite() {
-  const config = {
-    shape: {
-      dimension: {
-        // Set maximum dimensions
-        maxWidth: settings.sprite.width,
-        maxHeight: settings.sprite.width,
-      },
-      id: {
-        separator: settings.sprite.separator,
-      },
-      spacing: {
-        // Add padding
-        padding: 0,
-      },
-    },
-    mode: {
-      symbol: true, // Activate the «symbol» mode
-    },
-  };
-
```

Comment:
This config does not apply to the new package. As far as I can tell, this is not necessary as long as we continue to use 24x24 SVGs with `height`, `width`, and `viewBox` set properly.